static/js/components/ErrorMessage_test.js

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import { assert } from 'chai';
+import _ from 'lodash';
+
+import {
+  RECEIVE_DASHBOARD_FAILURE,
+  RECEIVE_GET_USER_PROFILE_SUCCESS,
+  REQUEST_GET_USER_PROFILE,
+  RECEIVE_DASHBOARD_SUCCESS,
+  RECEIVE_GET_USER_PROFILE_FAILURE,
+  SET_USER_PAGE_DIALOG_VISIBILITY,
+  START_PROFILE_EDIT,
+  UPDATE_PROFILE_VALIDATION,
+  REQUEST_PATCH_USER_PROFILE,
+  RECEIVE_PATCH_USER_PROFILE_FAILURE,
+  CLEAR_PROFILE_EDIT,
+} from '../actions';
+import {
+  DASHBOARD_RESPONSE,
+  ERROR_RESPONSE,
+  USER_PROFILE_RESPONSE,
+} from '../constants';
+import IntegrationTestHelper from '../util/integration_test_helper';
+import { makeStrippedHtml } from '../util/util';
+import * as api from '../util/api';
+import ErrorMessage from './ErrorMessage';
+
+describe("ErrorInfo", () => {
+  let errorString = `Sorry, we were unable to load the data necessary
+      to process your request. Please reload the page.`;
+  errorString = errorString.replace(/\s\s+/g, ' ');
+
+  let contactExpectation = `If the error persists, please contact 
+      mitx-support@mit.edu specifying this entire error message.`;
+  contactExpectation = contactExpectation.replace(/\s\s+/g, ' ');
+
+  describe('unit tests', () => {
+    let that;
+    let renderErrorMessage = props => {
+      return makeStrippedHtml(<ErrorMessage {...props} />);
+    };
+    let codeAttributes = [
+      ['error_code', '500'],
+      ['errorStatusCode', 404],
+    ];
+    let messageAttributes = [
+      ['user_message', 'A message'],
+      ['detail', 'Some details'],
+    ];
+
+    beforeEach(() => {
+      that = {
+        errorInfo: {
+          error_code: '500',
+          user_message: 'A message',
+          detail: 'Some details',
+          errorStatusCode: 404,
+        }
+      };
+    });
+
+    it('should take an errorInfo prop', () => {
+      let errorInfo = <ErrorMessage {...that}/>;
+      assert.deepEqual(errorInfo.props.errorInfo, that.errorInfo);
+    });
+
+    codeAttributes.forEach(([codeAttribute, code]) => {
+      messageAttributes.forEach(([msgAttribute, message]) => {
+        it(`should render an error and message on the ${codeAttribute} and ${msgAttribute} attributes`, () => {
+          let that = {
+            errorInfo: {
+              [codeAttribute]: code,
+              [msgAttribute]: message,
+            }
+          };
+          let errorMessage = renderErrorMessage(that);
+          assert.equal(true, errorMessage.includes(String(code)));
+          assert.equal(true, errorMessage.includes(errorString));
+          assert.equal(true, errorMessage.includes(contactExpectation));
+          assert.equal(true, errorMessage.includes(message));
+        });
+      });
+    });
+  });
+
+  describe('showing errors on pages', () => {
+    let renderComponent, helper, patchUserProfileStub, listenForActions;
+    let dashboardErrorActions;
+
+    const confirmErrorMessage = (div, expectedMessageText) => {
```

Comment:
Can you break `expectedMessageText` into two separate arguments instead of an array of two arguments? This should also allow you to use a default argument for the second argument